Project1_tr

```diff
@@ -0,0 +1,350 @@
+____________________________________________________________
+F     F     F                               T
+F     F     T                               T
+F     T     F                               T
+F     T     T                               T
+T     F     F                               T
+T     F     T                               T
+T     T     F                               T
+T     T     T                               T
+
+
+(b) (A→B)∧(A→¬B)
+Contingent 
+
+A     B     ((A → B) ∧ (A → ¬B))
+___________________________________
+F     F               T
+F     T               T
+T     F               F
+T     T               F
+
+
+(c) (A→(B∨C))∨(C→¬A) 
+Tautology
+
+A     B      C                ((A → (B ∨ C)) ∨ (C → ¬A))
+__________________________________________________________
+F     F      F                             T
+F     F      T                             T
+F     T      F                             T
+F     T      T                             T
+T     F      F                             T
+T     F      T                             T
+T     T      F                             T
+T     T      T                             T
+
+
+(d) ((A→B)∧C)∨(A∧D)
+Contingent
+
+A     B      C      D            (((A → B) ∧ C) ∨ (A ∧ D))
+____________________________________________________________
+F     F      F      F                           F
+F     F      F      T                           F
+F     F      T      F                           T
+F     F      T      T                           T
+F     T      F      F                           F
+F     T      F      T                           F
+F     T      T      F                           T
+F     T      T      T                           T
+T     F      F      F                           F
+T     F      F      T                           T
+T     F      T      F                           F
+T     F      T      T                           T
+T     T      F      F                           F
+T     T      F      T                           T
+T     T      T      F                           T
+T     T      T      T                           T
+
+
+
+2. A literal is an atomic formula or the negation of an atomic formula. We say a formula is in conjunctive normal form (CNF) if it is the conjunction of the disjunction of literals. Find propositional logic formulas in CNF equivalent to each of the following:
+
+(a) (A→B)→C
+
+  (A v C) ^ (~B v C)     CNF
+
+(b) (A→(BvC))v(C→~A)
+  
+  Already a tautology; CNF 
+  A v ~A
+
+(c) (¬A^¬B^C)v(¬A^¬C)v(B^C)vA
+  
+  Already a tautology; CNF  
+  A v ~A
+
+
+3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why:
+
+(a)  B(x,y) that says that x is a brother of y 
+  
+  Def.   ∃x∃y∃z(P(z,y)^P(z,x)^¬Fx^¬(y=x))
```

Comment:
The variables for B(x,y) should be under universal scope. As written, this definition of 'brother of' is satisfied by one pair, but a definition of 'brother of' should hold for all pairs that meet the conditions.